src/App.js

```diff
@@ -35,14 +36,20 @@ class App extends Component {
             state: 'contributions',
             withRefs: true
         });
+        if (!dbx.app) {
+            dbx.initialize();
+        }
     }
 
     render() {
         return (
             <div className="App">
                 <ButtonAppBar/>
                 <h1>My Contributions</h1>
-                <MainPageTB contributions={this.state.contributions}/>
+                <MainPageTB 
+                    contributions={this.state.contributions}
+                    dropbox={dbx}
```

Comment:
Don't pass singleton as prop, import in child component

Reply:
Done